multimodal/setup.py

```diff
@@ -41,7 +41,7 @@
     "smart_open>=5.2.1,<5.3.0",
     "pytorch_lightning>=1.7.4,<1.8.0",
     "text-unidecode<=1.3",
-    "torchmetrics>=0.8.0,<0.9.0",
+    "torchmetrics>=0.11.0,<=0.11.0",
```

Comment:
I think this upgrade has some API breakages. Need to investigate what metrics are affected. For example, `roc_auc` no longer supports `pos_label` in the new releases.

Reply:
Sure, also 0.11.0 didn't include PR that increase mAP evaluation speed. I'll copy the latest mAP implementation here instead of using the latest version.